### ISSUE TITLE
Waterfall Plot may not display legend

### ISSUE THREAD

**Issue by KarlPalmen** [mantidproject/mantid#19208]
### Expected behavior

Waterfall plot always shows its legend

### Actual behaviour

Waterfall plot can occur without legend being shown or more precisely it is shown then quickly removed.
Sometimes the legend is shown far up, so only its bottom is visible.

### Steps to reproduce the behaviour

Run script
```
base = CreateSampleWorkspace(Function='Powder Diffraction', NumBanks=4, BankPixelWidth=1)

wsNames = []

for t in range (10,110,10):
    wsName = "temp" + str(t)
    wsNames.append(wsName)
    MultiplyRange(base,75,99,t/100.0,OutputWorkspace=wsName)
    AddSampleLog(wsName,"Temp",str(t),"Number","Kelvin")

TestGroup = GroupWorkspaces(wsNames)
```
Then do waterfall plot of one spectrum of the TestGroup workspace.

### Platforms affected

Windows 7 at least.

**Comment by KarlPalmen** [comment]
I found that the legend was moved up almost out of view and could be moved back into view.

**Comment by KarlPalmen** [comment]
Hopefully will be fixed by change to Mantid 4.0
